### PR TITLE
Foot note conversion

### PR DESCRIPTION
**Pull request by kingthorin**
Per OWASP/OWASP-Testing-Guide-v5#10

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Converted previously set HTML footnotes to markdown per discussion in OWASP/OWASP-Testing-Guide-v5#10

### CODE REVIEW

**Comment by kingthorin** [comment]
I've removed the seemingly empty headings.

File: document/4 Web Application Security Testing/4.2 Information Gathering/4.2.1 Conduct search engine discovery reconnaissance for information leakage (OTG-INFO-001).md [hunk 1]
```diff
@@ -69,12 +69,6 @@ The Google Hacking Database is list of useful search queries for Google. Queries
 
 ## References
```
Comment (ThunderSon) [inline document/4 Web Application Security Testing/4.2 Information Gathering/4.2.1 Conduct search engine discovery reconnaissance for information leakage (OTG-INFO-001).md]: One last as you're at it and it's just about some stuff you cleaned up

Comment (kingthorin) [reply document/4 Web Application Security Testing/4.2 Information Gathering/4.2.1 Conduct search engine discovery reconnaissance for information leakage (OTG-INFO-001).md]: This one is actually good (I think). I've moved Tools to a level 3 to be within References, and moved the Remediation section above Refs.

**Comment by kingthorin** [comment]
Thanks @ThunderSon 

Do we really need another issue? Isn't this covered in #10 ?

**Pull request merged**